### PR TITLE
Update one-click_deploy.rst - Enabled USB debugging on android devices.

### PR DESCRIPTION
**Pull request by opsaaaaa**
The android icon did not show up in Godot until I enabled usb debugging on my phone.

### CODE REVIEW

File: tutorials/export/one-click_deploy.rst [hunk 1]
```diff
@@ -45,8 +45,8 @@ Support for more platforms such as iOS is planned.
 Using one-click deploy
 ----------------------
 
-- If deploying to Android, enable developer mode on the your mobile device then
-  connect the device to your PC using an USB cable.
+- If deploying to Android, enable developer mode on the your mobile device and turn on USB debugging. 
+- Then connect the device to your PC using an USB cable.
```
Comment (Calinou) [inline tutorials/export/one-click_deploy.rst:49]:
```suggestion
- If deploying to Android, enable developer mode on your mobile device then enable USB debugging.
- After enabling USB debugging, connect the device to your PC using an USB cable.
```

File: tutorials/export/one-click_deploy.rst [hunk 2]
```diff
@@ -45,8 +45,8 @@ Support for more platforms such as iOS is planned.
 Using one-click deploy
 ----------------------
 
-- If deploying to Android, enable developer mode on the your mobile device then
-  connect the device to your PC using an USB cable.
+- If deploying to Android, enable developer mode on your mobile device then enable USB debugging.
```
Comment (Calinou) [inline tutorials/export/one-click_deploy.rst:48]:
```suggestion
- If deploying to Android, enable developer mode on your mobile device then enable USB debugging in the device's settings.
```

**Comment by mhilbrunner** [comment]
Can confirm this is correct on all 3 current Android devices I tested.

Thank you and congrats on your first Godot docs contribution! 🎉

**Comment by Calinou** [comment]
Cherry-picked to the `3.3` and `stable` branches.

**Pull request merged**